test/Dialect/MSFT/partition.mlir

```diff
@@ -1,4 +1,3 @@
-// RUN: circt-opt %s --msft-partition -verify-diagnostics -split-input-file | FileCheck %s
```

Comment:
The output from "--msft-partition" without wire cleanup is sooo complex so as to make visual verification impossible, so testing it with filecheck is worthless given that I would just be copy and pasting it.

Reply:
I think its possible to write a meaningful FileCheck test for just a small part of the design... but testing both of these passes in conjunction sounds fine to me.

Reply:
Yeah, perhaps just one op tagged.